game_event/templates/game-events.html

```diff
@@ -36,6 +36,9 @@ <h1>Game Events</h1>
 				<th>Minimum Players</th>
 				<th>Maximum Players</th>
 				<th>Court ID</th>
+                <th>City</th>
```

Comment:
please fix the indentation here

Reply:
@SchSeba The indentation in this file is different and OK on my local repo, and I didn't make changes in the template, how is it possible?
<img width="778" alt="image" src="https://github.com/redhat-beyond/MeetBalls/assets/95130756/7d631234-0095-4ff0-b925-0e203c84b672">

Reply:
weird that is fine then..